beta-src/src/components/controllers/WDMapController.tsx

```diff
@@ -153,7 +97,7 @@ const WDMapController: React.FC = function (): React.ReactElement {
       dispatch(gameApiSliceActions.updateOrdersMeta(ordersMeta));
```

Comment:
This looks a little odd, I'm not sure the need to set a timeout.

Reply:
I don't know what's going on here. @bryancodazen , maybe you can comment?

Reply:
Since the drawing is based on ordersMeta getting updated, this runs updateOrdersMeta again on initial render to make sure orders are drawn (sometimes the map takes longer to load depending on external factors). Ideally this should not have a setTimeout and should just dispatch pure drawing of orders after initial render (territories and units are visible). It is something we will look into improving if time allows